common-content/en/module/induction/wrapping-up/index.md

```diff
@@ -20,12 +19,15 @@ emoji= '🧩'
 
 Now you'll need to **create a commit**.
 
-You can use this video 👉 https://www.youtube.com/watch?v=B8RSMBSzFuA to figure out how to create a commit.
+You can use the video below to figure out how to create a commit.
```

Comment:
Try to avoid above and below if you can, because spatial language presumes a way of interacting that might not always be true. (This is a tiny nitpick and nonblocking)